src/Handler/PropertyMapper.php

```diff
@@ -40,6 +40,10 @@ public function __invoke(
             $propertyInfo = $propertyMap->getProperty($key);
             $type = $propertyInfo->getType();
 
+			if ($propertyInfo->isNullable() && is_null($value)) {
+				continue;
+			}
```

Comment:
if for the example of `ComplexObject` y get this on the json:
```json
{
  child: null
}
```
and the attribute is nullable, skip the mapping.

Reply:
- Something is off with the indentation.
- I can think of a use case where the PHP model has a default value other than `null` meaning if the property is nullable and the value is null we explicitly set the null value.